src/pages/repair-request.tsx

```diff
@@ -0,0 +1,275 @@
+// Page for submitting a repair request
+
+import { ChangeEvent, useState } from "react";
+import { Inter } from "next/font/google";
+import Image from "next/image";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+
+import { useBrands } from "@/hooks/brands";
+import { useItemTypes } from "@/hooks/item-types";
+
+const inter = Inter({ subsets: ["latin"] });
+
+type FormValues = {
+  itemBrand: string;
+  itemType: string;
+  description: string;
+  images: [];
+  eventId: string;
+  tncAccepted: false;
+};
+
+const Home = () => {
+  const {
+    control,
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<FormValues>({
+    defaultValues: {
+      itemBrand: "",
+      itemType: "",
+      description: "",
+      images: [],
+      eventId: "",
+      tncAccepted: false
+    }
+  });
+
+  const itemTypeList = useItemTypes();
+  const brandList = useBrands();
+
+  // Brand dropdown
+  const [brand, setBrand] = useState("");
+  const handleBrandChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    setBrand(event.target.value);
+  };
+
+  // Item Types Dropdown
+  const [itemType, setItemType] = useState("");
+  const handleItemTypeChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    setItemType(event.target.value);
+  };
+
+  // Events Dropdown
+  const [event, setEvent] = useState("");
+  const handleEventChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    setEvent(event.target.value);
+  };
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    const response = await fetch(`/api/repair-request`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({
+        eventId: data.eventId,
+        itemBrand: data.itemBrand,
+        itemType: data.itemType,
+        description: data.description
+      })
+    });
+    ` `;
+
+    if (response.ok) {
+      alert("Data submitted");
+    } else {
+      alert(`Error! ${response.statusText}`);
+    }
+  };
+
+  return (
+    <main
+      className={`flex min-h-screen flex-col items-center ${inter.className}`}
+    >
+      <br></br>
```

Comment:
`<br />` tags are generally no longer used because they act as styling which should be the responsiblity of CSS. Opt for a CSS solution. You could use the `gap` class to space flexbox items.

```suggestion
      className={`flex min-h-screen flex-col items-center gap-4 p-4 ${inter.className}`}
```